hathor/nanocontracts/runner/runner.py

```diff
@@ -743,6 +743,7 @@ def syscall_get_rng(self) -> NanoRNG:
         """Return the RNG for the current contract being executed."""
         if self._rng is None:
             raise ValueError('no seed was provided')
+        return NanoRNG(seed=self._rng.randbits(256))
```

Comment:
This behavior will break compatibility with the current testnet if any blueprint is using RNG. If not, it's ok.